content/en/infrastructure/livecontainers.md

```diff
@@ -229,9 +253,19 @@ container_include: ["name:frontend.*"]
 [2]: /integrations/docker_daemon/
 [3]: /agent/kubernetes/
 [4]: /integrations/amazon_ecs/
-[5]: /agent/docker/#run-the-docker-agent
-[6]: /agent/docker/log/?tab=hostinstallation#activate-log-integrations
-[7]: /logs/live_tail/
-[8]: /getting_started/tagging/
-[9]: /getting_started/tagging/unified_service_tagging
-[10]: https://gist.github.com/hkaj/404385619e5908f16ea3134218648237
+[5]: https://app.datadoghq.com/containers/kubernetes-beta
+[6]: /agent/docker/#run-the-docker-agent
+[7]: /agent/kubernetes/?tab=helm
+[8]: /agent/
+[9]: /agent/cluster_agent/setup/?tab=secret
+[10]: /agent/kubernetes/?tab=helm#installation
+[11]: /getting_started/tagging/
+[12]: /getting_started/tagging/unified_service_tagging
+[13]: /dashboards/widgets/timeseries/
+[14]: /logs
+[15]: /metrics
+[16]: /tracing
+[17]: /events
+[18]: /logs/live_tail/
+[19]: https://gist.github.com/hkaj/404385619e5908f16ea3134218648237
+[20]: https://github.com/DataDog/helm-charts
```

Comment:
maybe use this link instead https://github.com/DataDog/helm-charts/tree/master/charts/datadog for a direct link to the configurable values